WaterNetworkManagement/Simulation/Result/schema.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "$id": "https://smart-data-models.github.io/data-models/specs/WaterNetwork/Result/schema.json",
+    "title": "WaterNetwork - Result",
+    "description": "This entity contains a harmonised description of a generic simulation result made for the Water Network Management domain. This entity is primarily associated with the water network management vertical and related IoT applications.",
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
+        },
+        {
+            "$ref": "https://raw.githubusercontent.com/smart-data-models/dataModel.WaterNetworkManagement/master/WaterNetworkManagement-schema.json"
+        },
+        {
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "Result"
+                    ],
+                    "description": "NGSI-LD Entity Type"
+                },
+                "description": {
+                    "$ref": "https://forge.etsi.org/rep/NGSI-LD/NGSI-LD/raw/master/schema/Entity.json#/definitions/Property"
+                },
+                "hasInputNetwork": {
+                    "$ref": "https://forge.etsi.org/rep/NGSI-LD/NGSI-LD/raw/master/schema/Entity.json#/definitions/Relatnioship"
+                },
+                "hasSimulation": {
+                    "$ref": "https://forge.etsi.org/rep/NGSI-LD/NGSI-LD/raw/master/schema/Entity.json#/definitions/Relatnioship"
+                },
+                "operationalControl": {
+                    "$ref": "https://forge.etsi.org/rep/NGSI-LD/NGSI-LD/raw/master/schema/Entity.json#/definitions/Property"
+                },
+                "outputParameters": {
+                    "$ref": "https://forge.etsi.org/rep/NGSI-LD/NGSI-LD/raw/master/schema/Entity.json#/definitions/Property"
+                },
+                "targetURI": {
```

Comment:
This doesn't capture the nesting of the properties (i.e. `targetURI` is a sub-relationship of the water attribute property) - should it?

Reply:
In the $ref, they define a field called "additionnalProperties" which will allow us to add a sub-property or a sub-relationship. `targetURI` is required that's true, but I would prefer simple schema.